cmd/docs/main_test.go

```diff
@@ -0,0 +1,39 @@
+package docs
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_DocsCommand(t *testing.T) {
+	testCases := []struct {
+		desc     string
+		expected []byte
+	}{
+		{
+			desc:     "Test datree docs",
+			expected: []byte(`Opening https://hub.datree.io in your browser.`),
+		},
+	}
+	for _, tC := range testCases {
+		t.Run(tC.desc, func(t *testing.T) {
+			cmd := New()
+			out := bytes.NewBufferString("Opening https://hub.datree.io in your browser.")
+			cmd.SetOut(out)
+			err := cmd.Execute()
```

Comment:
Okay, I will look into how I can mock `browser.OpenURL(url)` behaviour in the test.

Reply:
Hi @myishay, I have checked the way, but I can't find a way to mock. I also checked how the [browser](https://github.com/pkg/browser) library implemented test cases, but they did it the same way and it passes the test case in their CI/CD just like ours. 
If you have another idea we can mock, please let me know.

Reply:
Looking at it now

Reply:
> Hi @myishay, I have checked the way, but I can't find a way to mock. I also checked how the [browser](https://github.com/pkg/browser) library implemented test cases, but they did it the same way and it passes the test case in their CI/CD.  If you have another idea we can mock, please let me know.

I looked at the browser library. It seems like they don't have any tests:
<img width="468" alt="Screen Shot 2022-10-20 at 14 29 29" src="https://user-images.githubusercontent.com/22665577/196936793-2b72f090-ce2d-461c-bbd6-a73281048a5a.png">
By the way, you can have a look at the [tests workflow run](https://github.com/datreeio/datree/actions/runs/3287564623/jobs/5416874766) and see that it failed probably because it is a "headless machine".

Regarding mocking functions, there are 2 options for that which are described pretty good in the top answer of this [stack-overflow question](https://stackoverflow.com/questions/19167970/mock-functions-in-go).
The solution which we used in other commands for mocking is creating context for each command and then mocking this context in the test file. I think the easiest implementation in our code is [in the version command test case](https://github.com/datreeio/datree/blob/main/cmd/version/main_test.go).
It might not be the easiest to implement for the first time, so don't hesitate to ask questions.

Reply:
Thank you for showing me the way. I will try my best.